interface/patient_file/merge_patients.php

```diff
@@ -4,24 +4,28 @@
  * This script merges two patient charts into a single patient chart.
  * It is to correct the error of creating a duplicate patient.
  *
+ * @category  Patient_Data
  * @package   OpenEMR
- * @link      http://www.open-emr.org
  * @author    Rod Roark <rod@sunsetsystems.com>
  * @author    Brady Miller <brady.g.miller@gmail.com>
- * @copyright Copyright (c) 2013 Rod Roark <rod@sunsetsystems.com>
- * @copyright Copyright (c) 2018 Brady Miller <brady.g.miller@gmail.com>
- * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @copyright 2013 Rod Roark <rod@sunsetsystems.com>
+ * @copyright 2018 Brady Miller <brady.g.miller@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE
+ * GNU General Public License 3
+ * @link      http://www.open-emr.org
```

Comment:
lets keep the current header tags intact/standardized unless community decides to change them up